src/main/java/com/minecolonies/network/messages/PermissionsMessage.java

```diff
@@ -334,23 +355,30 @@ public void fromBytes(@NotNull ByteBuf buf)
         @Override
         public IMessage onMessage(@NotNull RemovePlayer message, @NotNull MessageContext ctx)
         {
+            ctx.getServerHandler().playerEntity.getServerWorld().addScheduledTask(() ->
+            {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00108)